ckanext/datastore/backend/postgres.py

```diff
@@ -1027,8 +998,11 @@ def upsert_data(context, data_dict):
     fields = _get_fields(context, data_dict)
     field_names = _pluck('id', fields)
     records = data_dict['records']
-    sql_columns = ", ".join(['"%s"' % name.replace(
-        '%', '%%') for name in field_names] + ['"_full_text"'])
+    sql_columns = ", ".join(
+        identifier(name) for name in field_names)
+    sql_full_text = " || ' ' || ".join(
```

Comment:
this looks like is not used or am i missing something ?

Reply:
good catch, this was some code from an earlier attempt that I forgot to delete